portfolio/src/main/webapp/script.js

```diff
@@ -129,29 +127,42 @@ function getOrder() {
  */
 function isLoggedIn() {
 
-  fetch('/login').then(response => response.json()).then((status) => {
-    let loginStatus = status[0];
+  fetch('/login').then(response => response.json()).then((stat) => {
```

Comment:
[Nit] Why did you change 'status' to 'stat'?  I think 'status' is more readable?

Reply:
status was marked as a keyword by VIM, so I changed it to avoid that. However, it seemed to be working fine before, so I will switch back and test again.

Reply:
It does appear to be reserved
https://www.w3schools.com/JS/js_reserved.asp

In that case, I would rather use a longer name instead of a shorter one.  Also, it appears that this variable stores all the JSON info so maybe call it something like 'jsonUserInfo'